examples/FsCheck.XUnit.CSharpExamples/ReverseFixture.cs

```diff
@@ -6,17 +6,25 @@ namespace FsCheck.XUnit.CSharpExamples
 {
     public class ReverseFixture
     {
-        //[Property]
-        //public void Bcl2()
-        //{
-        //}
-
-        [Property]
+        [Property(QuietOnSuccess = true)]
         public bool Bcl(int[] xs)
         {
           return xs.Reverse().Reverse().SequenceEqual(xs);
         }
 
+        [Property(QuietOnSuccess = true)]
+        public void Bcl2(int[] xs)
```

Comment:
FWIW, when I originally developed the *QuietOnSuccess* feature, I added a similar 'test' to the F# tests. There was no automated feedback (because, in general, there was no test coverage of FsCheck.Xunit), but I used the test for the documentation to ensure that it at least compiles. I think that 'test' is still around.